Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -11,44 +11,31 @@ import UIKit
 import UserNotifications
 import Firebase
 
-extension UNAuthorizationStatus {
-    var hasPermission: Bool {
-        self == .authorized || self == .provisional
-    }
-}
+class RemoteNotificationHandler {
 
-extension UNAuthorizationStatus: CustomStringConvertible {
+    @UserDefault(key: "io.gnosis.multisig.deviceID")
+    private var storedDeviceID: String?
 
-    public var description: String {
-        switch self {
-        case .authorized: return "Authorized (granted)"
-        case .denied: return "Denied"
-        case .notDetermined: return "Not Determined"
-        case .provisional: return "Provisional (granted)"
-        @unknown default: return "Unknown: \(rawValue)"
-        }
-    }
-}
+    @EnumDefault(key: "io.gnosis.multisig.authorizationStatus")
+    private var authorizationStatus: UNAuthorizationStatus?
```

Comment:
Why do we need to store authorizationStatus in user defaults?

Reply:
this is to monitor if the status have changed, and if yes and it was granted, to register the safes again